include/eve/module/arithmetic/regular/impl/bit_floor.hpp

```diff
@@ -8,18 +8,19 @@
 #pragma once
 
 #include <eve/detail/implementation.hpp>
+#include <eve/constant/one.hpp>
+#include <eve/constant/zero.hpp>
+#include <eve/function/bit_cast.hpp>
+#include <eve/function/bit_shr.hpp>
 #include <eve/function/bit_width.hpp>
 #include <eve/function/dec.hpp>
-#include <eve/function/ifrexp.hpp>
 #include <eve/function/if_else.hpp>
+#include <eve/function/ifrexp.hpp>
+#include <eve/function/is_eqz.hpp>
 #include <eve/function/is_less.hpp>
-#include <eve/function/is_less_equal.hpp>
 #include <eve/function/ldexp.hpp>
-#include <eve/function/max.hpp>
-#include <eve/constant/one.hpp>
 #include <type_traits>
 #include <eve/concept/value.hpp>
-#include <eve/detail/apply_over.hpp>
```

Comment:
apply_over is still use, why remove the include ?

Reply:
reinserted